### PR TITLE
Lps 118086 crypto

### PR DESCRIPTION
**Pull request by arthurchan35**
Hey @Brianchandotcom
This is the initial pull to Crypto Hash Library, containing only the APIs.
We have SPIs and Impls ready but they will come later once this pull is merged, so each pull is relative small and easy to merge.

### CODE REVIEW

File: modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+hash-pepper-storage-select=Hash Pepper Storage OSGi Target Filter Selector
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-api/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:sf

**Comment by arthurchan35** [comment]
ci:forward

**Pull request closed without merging**